types/react-native-dotenv/tsconfig.json

```diff
@@ -3,7 +3,7 @@
         "module": "commonjs",
         "lib": ["es6"],
         "esModuleInterop": true,
-        "noImplicitAny": true,
+        "noImplicitAny": false,
```

Comment:
This still seems to be changed?

Reply:
Umm ok weird, here it should be fixed now.